components/insight/SRC/org/openmicroscopy/shoola/util/ui/slider/TwoKnobsSliderModel.java

```diff
@@ -280,7 +280,7 @@ void setStartValue(double startValue)
 
 	/**
 	 * Sets the value of the end knob, value in the range
-	 * <code>]{@link #startValue}, {@link #maximum}]</code>.
+	 * <code>]{@link #startValue}, {@link #maximum}]  (one based) </code>.
```

Comment:
@jburel: I'm used to the http://mathworld.wolfram.com/Interval.html notation but that might just be coincidence with what I happen to have been exposed to!